.github/workflows/integration-tests-1.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Integration Tests 1
+on:
+  workflow_run:
+    workflows: ["Verify Build"]
+    branches: ["main"]
+    types:
+      - completed
+  workflow_dispatch:
+
+env:
+  GOPATH: /opt/go
+  PATH: /opt/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  GO_VER: 1.18.2
+
+jobs:
```

Comment:
Is it possible to have a single integration test workflow that runs 5 jobs? It appears that you can now re-trigger individual jobs that have failed in a workflow - https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs#re-running-a-specific-job-in-a-workflow

Reply:
The one advantage to having each integration test in their own workflow is the ability to manually trigger any one of them at any time through the Actions tab. If we only care about re-triggering workflows when they have failed, then yes we can just have one workflow that runs 5 jobs.

Reply:
My opinion is that there is not much benefit to trigger them individually from the Actions tab. Typically if you want to trigger the integration tests from UI you'd want to run them all.

The split into 5 sets is somewhat arbitrary, so it doesn't feel right to make them 5 first class workflows that need to be maintained individually. I'd prefer one workflow to trigger them all while still benefitting from running them in parallel and ability to re-run any one that failed. So for me, that implies one workflow with 5 parallel jobs.